rules/windows/credential_access_credential_dumping_msbuild.toml

```diff
@@ -23,9 +23,9 @@ tags = ["Elastic", "Windows"]
 type = "query"
 
 query = '''
-event.category:process and event.type:change and
+event.category:library and event.type:change and
   (winlog.event_data.OriginalFileName:(vaultcli.dll or SAMLib.DLL) or
-    dll.name:(vaultcli.dll or SAMLib.DLL)) and
+    dll.name:(vaultcli.dll or SAMLib.DLL) or file.name:(vaultcli.dll or SAMLib.DLL)) and
   process.name: MSBuild.exe
```

Comment:
I don't think this is compatible with winlogbeat and endpoint (may not have been in its original state either).

Reply:
agree, will delete this change from this PR, need to be reworked.